package Day62;

public class Test  {

	 

	public static void main(String[ ] args) {

		int[ ] arr = { 1, 2, 3};

		try {

		System.out.println( arr[100] );

		} catch(RuntimeException  e) {

		System.out.println("Runtime Exception Found");

		} catch (ArrayIndexOutOfBoundsException e) {

		System.out.println("Array index out of bound Exception Found");

		} catch (IndexOutOfBoundsException e) {

		System.out.println("Index out of bound Exception Found");

		}

		}
}


